pysages/methods/ann.py

```diff
@@ -200,7 +200,7 @@ def learn_free_energy(state):
         nn = train(state.nn, phi)
         params = pack(nn.params, layout)
         phi = nn.std * model.apply(params, inputs).reshape(phi.shape)
-        phi = phi - phi.min()
+        phi = phi.max() - phi
```

Comment:
I think this needs should be kept as it was. The only change needed should be the one below.